src/recipes/Viewer.tsx

```diff
@@ -0,0 +1,279 @@
+    } = this.state;
+
+    return (
+      <View style={{ width: "100%" }}>
+        <InfoCard title="Current Owner">
+          <View style={styles.container}>
+            <View style={styles.rowLeft}>
+              <CircularButton
+                style={styles.circButton}
+                label={nI(owner.name)}
+                color={colors.white.base}
+                backgroundColor={colors.blue.base}
+              />
+              <View style={{ flex: 1 }}>
+                <Text color={colors.gray.darker} size={15}>
+                  {this.isUser(owner.id) ? "You" : owner.name}
+                </Text>
+                {owner.subText}
+              </View>
+
+              {this.isUser(owner.id) ? (
+                <Select
+                  options={viewers}
+                  disabled={disabled}
+                  ref={this.selectRef}
+                  label={() => (
+                    <Text
+                      color={
+                        disabled ? colors.violet.light : colors.violet.base
+                      }
+                    >
+                      Transfer Ownership
+                    </Text>
+                  )}
+                  onSelect={({ id }) =>
+                    this.setState({
+                      selectedAgentId: id
+                    })
+                  }
+                  placeholder="Choose whom to transfer"
+                  selected={selectedAgentId}
+                  showFooterButton
+                  autoClose={false}
+                  footer={
+                    <View
+                      style={{
+                        paddingHorizontal: 25,
+                        paddingBottom: 25
+                      }}
+                    >
+                      <Button.DoubleFooterButton
+                        leftButtonLabel={"Transfer & Follow"}
+                        rightButtonLabel={"Transfer"}
+                        onLeftButtonPress={() => {
+                          this.selectRef.current.toggle();
+                          this.openTransferAndFollowConfigurationModal();
+                        }}
+                        onRightButtonPress={() => {
+                          this.selectRef.current.toggle();
+                          this.openTransferConfirmationModal();
+                        }}
+                        leftDisabled={!selectedAgentId}
+                        rightDisabled={!selectedAgentId}
+                      />
+                    </View>
+                  }
+                />
+              ) : (
+                <Icon
+                  name="phone-filled"
+                  color={colors.violet.base}
+                  size={18}
+                  onPress={() => onCall(owner.phone)}
+                />
+              )}
+            </View>
+          </View>
+
+          {viewers && (
+            <InfoCard style={viewerInfoCard} title="Viewers">
+              {viewers.map(viewer => (
```

Comment:
`viewers.map( {id,name,phone}) => (
)`

id and name are the only 2 used below.